changelogs/current.yaml

```diff
@@ -29,6 +29,13 @@ removed_config_or_runtime:
     Removed runtime flag ``envoy.restart_features.allow_client_socket_creation_failure`` and legacy code paths.
 
 new_features:
+- area: spiffe
+  change: |
+    Added :ref:`trust_bundles <envoy_v3_api_field_extensions.transport_sockets.tls.v3.SPIFFECertValidatorConfig.trust_bundles>`
+    to the SPIFFE certificate validator configuration. This field allows specifying a SPIFFE trust bundle map as a
+    local JSON file through a data source.
+  This is an alternative to :ref:`trust_domains <envoy_v3_api_field_extensions.transport_sockets.tls.v3.SPIFFECertValidatorConfig.trust_domains>`.
+    The trust_bundle_map is mutually exclusive with trust_domains.
```

Comment:
I think there is a format problem? not sure. but you may need additional two spaces before the `This is an alternative.....`